src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -3303,6 +3303,21 @@ def create_lb_inbound_nat_rule(
     return get_property(poller.result().inbound_nat_rules, item_name)
 
 
+# workaround for : https://github.com/Azure/azure-cli/issues/17071
+def lb_get(client, resource_group_name, load_balancer_name):
+    lb = client.get(resource_group_name, load_balancer_name)
+    return lb_get_operation(lb)
+
+
+# workaround for : https://github.com/Azure/azure-cli/issues/17071
+def lb_get_operation(lb):
+    for item in lb.frontend_ip_configurations:
+        if item.zones is not None and len(item.zones) >= 3:
+            item.zones = None
```

Comment:
If user set zones [1, 2, 3] on purpose, the output of zones will be None?

Reply:
I have a test and it does not influence.

Reply:
Can user set zones as [1, 2, 3] in current version? If they can what's the response now?

Reply:
(1)'zones' don't support input more than 1